src/routes/api/playlists/library/+server.ts

```diff
@@ -0,0 +1,21 @@
+import { readPath } from '$lib/server/firebaseUtils';
+import { json } from '@sveltejs/kit';
+import type { RequestHandler } from './$types';
+import { filterAttributes } from '$lib/utils';
+
+export const GET = (async (request) => {
+  const attributes = request.url.searchParams.get('attributes')?.split(',') ?? [];
+  const playlists = (await readPath<Database.Playlists.Library>('/playlists/library')) || {};
+  const playlistArray = Object.values(playlists).map((playlist) => ({
+    ...playlist,
+    words: playlist.words?.map((word) => word.map((letters) => (letters === false ? null : letters))) ?? [],
```

Comment:
We should probably make a helper function for this since we do it now in multiple places

Reply:
yeah i thought about that and forgot to do it haha, doing it now!